pkg/didcomm/protocol/issuecredential/states.go

```diff
@@ -75,7 +75,16 @@ func (s *start) Name() string {
 }
 
 func (s *start) CanTransitionTo(st state) bool {
-	return st.Name() == stateNameProposalSent || st.Name() == stateNameProposalReceived
+	switch st.Name() {
+	// Issuer
```

Comment:
we should consider specifying and persisting the role the participants in a protocol are fulfilling

Reply:
The role can be determined by message type and inbound/outbound value. Do you mean to include this check/condition?

Reply:
let's defer that for later